web-server/plugins/slycat-dac/js/dac-scatter-buttons.js

```diff
@@ -33,16 +34,23 @@ var fisher_pos = null;
 var diff_button_used = false;
 var diff_desired_state = null;
 
+// is lof button out of date?
+// true if correct, false if out of date
+var lof_button_state = null;
```

Comment:
You should initialize this to true or false. null is basically a falsy value anyway so you could pro just set this to false.